CMakeLists.txt

```diff
@@ -147,8 +143,8 @@ set(CPACK_PACKAGE_VERSION_PATCH "${VERSION_PATCH}")
 set(CPACK_PACKAGE_VENDOR "Advanced Micro Devices Inc.")
 set(CPACK_PACKAGE_CONTACT "ROCM Validation Suite Support <rocm-validation-suite.support@amd.com>")
 #set(CPACK_PACKAGE_FILE_NAME "${CMAKE_PROJECT_NAME}-${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH}")
-set(CPACK_DEBIAN_PACKAGE_DEPENDS "rocblas")
-set(CPACK_RPM_PACKAGE_REQUIRES "rocblas")
+set(CPACK_DEBIAN_PACKAGE_DEPENDS "rocblas, libyaml-cpp-dev")
+set(CPACK_RPM_PACKAGE_REQUIRES "rocblas, yaml-cpp-devel")
```

Comment:
These changes are largely untested because I don't have rocblas in all of my docker images.  
From a quick check - yaml-cpp-devel is present on rhel 8.x, 9.x, and opensuse tumbleweed. libyaml-cpp-dev is present on ubuntu 22.04.